data_selection/static/data_selection.js

```diff
@@ -94,22 +103,97 @@ $('.sample-label span').on('click', function() {
   loadAudioSample(this.id);
   $('.sample-label span').removeClass('active');
   $('#' + id).addClass('active');
+
+  // Hide the result panel when the user click on another audio sample
+  $('#results-section').addClass('display-none');
 });
 
+// Restart the audio sample (return to the start)
 $('#go-start').on('click', function(){
   wavesurfer.stop();
   $('#stop-btn').addClass('display-none');
   $('#play-btn').removeClass('display-none');
 });
 
+// Finish the audio sample
 $('#go-end').on('click', function(){
   wavesurfer.skipForward(10);
   $('#stop-btn').addClass('display-none');
   $('#play-btn').removeClass('display-none');
 });
 
+let input = '';
+
+$('#input-transcription').keypress(function (e) {
+  if (e.which === 13) {
+  isLoading();
+  submit(input);
+  clearResults();
+  $('#results-section').addClass('display-none');
+      return false;
+  }}).on('input', function () {
+  input = this.value;
+  checkCompletion();
+});
+
+$('#submit-button').on('click', function(){
+  isLoading();
+  submit(input);
+  clearResults();
+  $('#results-section').addClass('display-none');
+});
+
+function submit(input) {
+
+  let currentSampleId = $('.active').attr('id');
+  var url = 'http://34.240.154.213:6121/api/compute/sim/' + uttId[currentSampleId];
+  var data = {"input": input};
+
+  console.log(input);
+  $.ajax({
+      type: "POST",
+      url: url,
+      contentType: 'application/json',
+      data: JSON.stringify(data, null, '\t'),
+      success: function (data) {
+
+      let jsonPhonSim = data['ng_json'];
```

Comment:
Indentation

Reply:
Also I already asked Yu to remove '_json' from the key because we know it's json. Check with him to change this